overlord/snapstate/handlers.go

```diff
@@ -838,7 +838,41 @@ func (m *SnapManager) doDiscardSnap(t *state.Task, _ *tomb.Tomb) error {
 	return nil
 }
 
-// aliases v2
+/* aliases v2
+
+aliases v2 implementation uses the following tasks:
+
+  * for install/refresh/remove/enable/disable etc
+
+    - remove-aliases: remove aliases of a snap from disk and mark them pending
+
+    - setup-aliases: (re)creates aliases from snap state, mark them as
+      not pending
+
+    - set-auto-aliases: updates aliases snap state based on the
+      snap-declaration and current revision info of the snap
```

Comment:
Thanks for the great comment here!

This is the only one that made me wonder in terms of naming. Should we go with something like "refresh-auto-aliases"? Do we have any precedents in other tasks? Actually, I see we have "refresh-aliases" below which feels very related. Would it make sense to merge them?

Everything else above and below feels very clear.